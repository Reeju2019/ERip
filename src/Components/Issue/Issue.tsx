import ExtraHeader from '../Extra/ExtraHeader'
import Header from '../Home/Header/Header'
import IssueHeader from './IssueHeader/IssueHeader'
import ProductDetails from './ProductDetails/ProductDetails'

const Issue: React.FunctionComponent = () => {
  return (
    <>
      <Header />
      <ExtraHeader/>
      <IssueHeader />
      <ProductDetails/>
    </>
  )
}

export default Issue
