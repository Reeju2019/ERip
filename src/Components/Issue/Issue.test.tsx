import { render} from '@testing-library/react'
import renderer from 'react-test-renderer'
import ReactDOM from 'react-dom'
import Issue from './Issue'

test('issue', () => {
  render(<Issue />)
})
describe('App rendering specification', () => {
  it('SnapShot testing', () => {
    const component = renderer.create(<Issue />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('render without crashing', () => {
    const div = document.createElement('div')
    render(<Issue />)
    ReactDOM.unmountComponentAtNode(div)
  })
})
jest.mock('react-router-dom', () => ({                                                                                                                                                                                                                                   
  ...jest.requireActual('react-router-dom'),
  useNavigate: () => jest.fn(),
}))
