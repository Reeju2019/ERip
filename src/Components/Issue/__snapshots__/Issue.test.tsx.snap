// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App rendering specification SnapShot testing 1`] = `
<div
  className="loading"
>
  <div />
</div>
`;
